lib/sisimai/bite/email/imailserver.rb

```diff
@@ -0,0 +1,181 @@
+module Sisimai::Bite::Email
+  module IMailServer
+    # Sisimai::Bite::Email::IMailServer parses a bounce email which created by
+    # Ipswitch IMail Server.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite::Email/IMailServer.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'x-mailer' => %r/\A[<]SMTP32 v[\d.]+[>][ ]*\z/,
+        :'subject'  => %r/\AUndeliverable Mail[ ]*\z/,
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->